src/lib/register-service-worker.js

```diff
@@ -120,7 +120,8 @@ const executeServiceWorker = (releaseHash: string, serviceWorker: string) => {
 }
 
 export function registerServiceWorker({ dumbledoreCurrentReleaseHash, dumbledoreServiceWorker }: RegisterServiceWorkerParams) {
-    if ('serviceWorker' in navigator === false) {
+    // Browser eligibility criteria for SW registration
+    if ('serviceWorker' in navigator === false || 'BroadcastChannel' in window === false) {
         getLogger().info(`${ LOG_PREFIX }NOT_SUPPORTED`);
```

Comment:
should we log which features were available?

```suggestion
        getLogger().info(`${ LOG_PREFIX }NOT_SUPPORTED`, {
          serviceWorker: Boolean('serviceWorker' in navigator),
          BroadcastChannel: Boolean('BroadcastChannel' in window),
        });
```